README.md

```diff
@@ -122,12 +118,13 @@ of `aicsimageio` while using `napari` please install
 
 ## Performance Considerations
 * **If your image fits into memory and you are not using a distributed cluster:** use
-`AICSImage.data` or `Reader.data` which are generally optimal.
+`AICSImage.data` or `Reader.data` which are generally optimal. You can also use this to
+preload the data before using `get_image_data`.
 * **If your image is too large to fit into memory:** use `AICSImage.get_image_data` to
 get a `numpy` array or `AICSImage.get_image_dask_data` to get a `dask` array for a
 specific chunk of data from the image.
-* **If you are using a distributed cluster:** all functions and properties in the
-library are generally optimal.
+* **If you are using a distributed cluster with more than ~6 workers:** all functions
```

Comment:
lines 123-125 need update now.  if image is too large, you basically have to use dask despite the appearance of get_image_data.

Reply:
These changes should be made on `master` and merged in to this branch. But agree!

Reply:
Fully removed performance considerations

Reply:
Resolved by #149